sdk/tracing/azure-core-tracing-opencensus/pom.xml

```diff
@@ -6,13 +6,13 @@
   <parent>
     <groupId>com.azure</groupId>
     <artifactId>azure-client-sdk-parent</artifactId>
-    <version>1.5.0</version>
+    <version>1.6.0</version>
     <relativePath>../../../pom.client.xml</relativePath>
   </parent>
 
   <groupId>com.azure</groupId>
   <artifactId>azure-core-tracing-opencensus</artifactId>
-  <version>1.0.0-preview.3</version>
+  <version>1.0.0-preview.4</version>
```

Comment:
wait for this to be released, before merging.

Reply:
Yes, I will wait until all releases are done before merging this.

Reply:
Yes, I will wait until all releases are done.